src/VisualStudio/Core/Def/Implementation/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -284,19 +290,47 @@ public int ClearErrors()
             int originalStartLine,
             int originalStartColumn,
             int originalEndLine,
-            int originalEndColumn)
+            int originalEndColumn,
+            DiagnosticAnalyzerInfoCache analyzerInfoCache)
         {
+            string title, description, category, helpLink;
+            DiagnosticSeverity defaultSeverity;
+            bool isEnabledByDefault;
+            ImmutableArray<string> customTags;
+
+            if (analyzerInfoCache != null && analyzerInfoCache.TryGetDescriptorForDiagnosticId(errorId, out var descriptor))
+            {
+                title = descriptor.Title.ToString(CultureInfo.CurrentCulture);
+                description = descriptor.Description.ToString(CultureInfo.CurrentCulture);
```

Comment:
https://github.com/dotnet/roslyn/issues/54557 was a similar issue.

```suggestion
                title = descriptor.Title.ToString(CultureInfo.CurrentUICulture);
                description = descriptor.Description.ToString(CultureInfo.CurrentUICulture);
```